### PR TITLE
(fix) Move explanations of managed service provider

### PR DESCRIPTION
**Pull request by mingoland**
## Trello card URL:
https://trello.com/c/Bbf7kMjS/262-explain-what-a-managed-service-provider-is

## Changes in this PR:
- Remove sidebar of managed service provide explanation from the "What are you looking for?" page
- Add hint to the question
- Add hints to the radio buttons

### Before
screenshot 2018-11-28 at 17 00 45

screenshot 2018-11-28 at 17 01 02

### After
screenshot 2018-11-28 at 17 00 30

screenshot 2018-11-28 at 17 00 14

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -262,16 +262,10 @@ en:
       managed_service_provider:
         answer_master_vendor: Master vendor managed service
         answer_neutral_vendor: Neutral vendor managed service
+        option1_hint: These supply whatever candidates they can. Where they don’t have a suitable candidate themselves, they find other suppliers who do.
```
Comment (chrisroos) [inline config/locales/en.yml]: Can you rename this to `master_vendor_hint` to mirror the names used above, please?

File: config/locales/en.yml [hunk 2]
```diff
@@ -262,16 +262,10 @@ en:
...
+        option2_hint: A neutral vendor only provides candidates from other suppliers - they have no candidates of their own.
```
Comment (chrisroos) [inline config/locales/en.yml]: Can you rename this to `neutral_vendor_hint` to mirror the names used above, please?

File: config/locales/en.yml (hunk 1 above)
Comment (mingoland) [reply config/locales/en.yml]: Changed to `master_vendor_hint`

File: config/locales/en.yml (hunk 2 above)
Comment (mingoland) [reply config/locales/en.yml]: Changed to `neutral_vendor_hint`

**Pull request merged**